azure-pipelines-ci.yml

```diff
@@ -14,22 +14,28 @@ jobs:
   strategy:
     matrix:
       Win32:
-        BuildPlatform: 'Win32'
+        OptionalBuildArch: ''
+        VcpkgTargetTriplet: 'x86-windows-static'
       Win64:
-        BuildPlatform: 'x64'
+        OptionalBuildArch: ' Win64'
+        VcpkgTargetTriplet: 'x64-windows-static'
 
   steps:
   - template: azure-pipelines/templates/ci-git-rebase.yml
   - template: azure-pipelines/templates/windows-dependencies.yml
   - template: azure-pipelines/templates/ci-opengfx.yml
+    parameters:
+      SharedFolder: C:/Users/Public/Documents/OpenTTD
   - template: azure-pipelines/templates/windows-build.yml
     parameters:
-      BuildPlatform: $(BuildPlatform)
-  - script: |
-      call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x86
-      cd projects
-      call regression.bat
+      OptionalBuildArch: $(OptionalBuildArch)
+      VcpkgTargetTriplet: $(VcpkgTargetTriplet)
+  - task: VSBuild@1
     displayName: 'Test'
+    inputs:
+      solution: build/RUN_TESTS.vcxproj
+      configuration: 'Release'
```

Comment:
I don't understand how regression is still working, because openttd.exe is now in build/RelWithDebInfo, but regression runs in build/Release